### ISSUE TITLE
any way to send the missing socket update to client after network restore

### ISSUE THREAD

**Issue by UsmanShafaqat007** [SocketCluster/socketcluster#585]
due to the network being down the socket got disconnected at the client's end and the client missed the update during that time.
is there any way to send that updates again when the socket connection is restored?

**Comment by maarteNNNN** [comment]
Let me rephrase, you *can't* rely on `socket.id`. You *will* have to setup ids of your own on your clients, on connection you *will* pass them on to the server and then you can track what's their state (connected or disconnected). I don't think this will be easy to implement.

I suggest that you instead store the messages on the server and use `.invoke('latest')` on the client and setup a route `.procedure('latest')` that provides the latest updates.

Make sense?
